site/en/blog/third-party-scripts/index.md

```diff
@@ -0,0 +1,264 @@
+
+3.  **Network**: Since third-parties are hosted on different origins, browsers have to make a larger number of connections to download content from them. The different connections cannot coordinate download based on priority, resulting in network contention and suboptimal network use. This can further delay the page load if [the proper optimizations](https://web.dev/preconnect-and-dns-prefetch/#:~:text=Adding%20rel%3Dpreconnect%20to%20a,time%20the%20browser%20requests%20them.) are not considered.
+
+4.  **Sequencing**: Third-parties can block the main thread and contend with bandwidth for more critical resources. That said, in some cases, third-parties are the critical resources required for rendering the page. Optimal sequencing of the first and third party resources becomes necessary when websites use multiple third-parties. Web developers should be aware of the different [options](https://web.dev/script-component/#sequencing-third-party-scripts-without-a-framework-component) available to optimize sequencing.
+
+As a consequence of the above, third-parties can affect any or all components of Core Web Vitals. A majority of third-parties negatively impact LCP and FID. [YouTube embeds block the main thread](https://almanac.httparchive.org/en/2021/third-parties#fig-15) for 4.5 seconds for 10% of the websites on mobile, and at least 1.6 seconds for 50% of the websites studied. Imagine a user's frustration if they came across a page with 20 such scripts on a slow connection. The following visualization from [thirdpartyweb.today](https://www.thirdpartyweb.today/) shows third-parties with the largest performance impact at present.
+
+{% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/U26McnxB3ZYKJ2zE0Zp4.png", alt="Third Party Web visualization", width="800", height="500" %}
+
+"Across top ~4 million sites, ~2700 origins account for ~57% of all script execution time with the top 50 entities already accounting for ~47%". - [third-party-web](https://github.com/patrickhulce/third-party-web)
+
+Pages that render quickly and become interactive within a reasonable timeframe are essential to a good user experience as measured by the CWV. Good UX often translates to good business for websites, which can mean good business for third-parties being used. Working together to reduce the impact of third-parties can be a win for everyone in the chain.
```

Comment:
```suggestion
Pages that render quickly and become interactive within a reasonable timeframe are essential to a good user experience as measured by the Core Web Vitals. Good UX often translates to good business for websites, which can mean good business for third-parties being used. Working together to reduce the impact of third-parties can be a win for everyone in the chain.
```

Reply:
"CWV" is an internal and informal term. We recommend avoiding this initialism in public-facing resources.